assets/tests/components/routePicker.test.tsx

```diff
@@ -34,6 +34,19 @@ describe("RoutePicker", () => {
     expect(tree).toMatchSnapshot()
   })
 
+  test("clicking the collapse button hides the route picker", () => {
+    const wrapper = mount(<RoutePicker routes={null} selectedRouteIds={[]} />)
+    expect(wrapper.find(".m-route-picker").hasClass("visible")).toBeTruthy()
+    expect(wrapper.find(".m-route-picker").hasClass("hiddden")).toBeFalsy()
+
+    act(() => {
+      wrapper.find(".m-route-picker__tab-button").simulate("click")
+    })
+
+    expect(wrapper.find(".m-route-picker").hasClass("visible")).toBeFalsy()
+    expect(wrapper.find(".m-route-picker").hasClass("hidden")).toBeTruthy()
+  })
```

Comment:
I think a snapshot test for a RoutePicker in the hidden state would be useful. (`mockUseStateOnce` might be useful for writing that, I'm not sure.)

Reply:
I don't think this would show anything other than the new class name. Is there something else I'm missing?